server/lib/data_import/my_anime_list.rb

```diff
@@ -31,15 +32,28 @@ def get_media(external_id) # anime/1234 or manga/1234
       end
     end
 
+    def get_character(external_id)
+      media = Mapping.lookup('myanimelist', external_id)
+      media ||= Character.new
+
+      get(MY_ANIME_LIST_HOST, external_id) do |response|
+        character = Extractor::Character.new(response, external_id)
+
+        media.assign_attributes(character.to_h.compact)
+
+        yield media
+      end
+    end
+
     private
 
-    def get(url, opts = {})
-      super(build_url(url), opts)
+    def get(host, url, opts = {})
+      super(build_url(host, url), opts)
     end
 
-    def build_url(path)
+    def build_url(host, path)
       return path if path.include?('://')
```

Comment:
Technically you could already have done `get("#{MY_ANIME_LIST_HOST}/#{external_id}")`, but I like the added `host` parameter.  Anyways, I'd recommend removing this line, since it will override the `host` param.  For example:

```ruby
build_url('http://hummingbird.me', 'http://myanimelist.net/anime/dickbutt')
# => 'http://myanimelist.net/anime/dickbutt'
```

It's an unexpected outcome, and I don't think this short-circuit is used anywhere.